### ISSUE TITLE
ObsidianX Distribution Proposal

### ISSUE THREAD

**Issue by dangershony** [obsidianproject/Obsidian-StratisNode#1]
## ObsidianX (ODX)  
ODX is a coin that will replace (eventually) the Obsidian (ODN) coin, due to several reasons it was decide to create a new network (reset the current ODN network) and airdrop ODN UTXOs to the new ODX coin.

ODN current distribution can be found here 
https://blockexplorer.obsidianproject.org/address/top

`34,957,487` ODN are the foundation coins address `XUsQeLmaMtcpNJ91e4BgdSHkFoGNv3zkdC`  
`10,220,235` ODN are believed to be the cryptopia lost coins address `XCosbm19z5uRzd1LHgVSyH9wW1wyJrD8gc`

Total supply is `108177206.35880000` ODN
https://blockexplorer.obsidianproject.org/api/getinfo  
(Note there is a bug in the total supply api endpoint blacky will fix it soon)

The ODN network will be maintained until the airdrop (not the snapshot date) 
We will take the last ODN block hash (snapshot block) and push it in to the genesis of ODX.

## ODX Distribution Proposal:  

**Changing the Block time:**  
Current ODN blocks time is `64` seconds.
I propose to reduce the block time to be `3-5` min (some code experimentation is needed here).
There is no benefit in more TPS as ODX is not going to be a payment network but a network to support the Decentralized Messenger. 

**Reward system proposal:**  
Current reward is `20` ODN

To reward the coin holders that are still active I propose the airdrop will be a 2:1 ratio, this means for every 2 ODN 1 ODX will be airdropped, such an airdrop will make the total supply to be about `54,088,603` coins.
To make up for the burning of coins the reward will grow for two years.

For a period of `350,400` blocks (or 48 month assuming 3 min blocks) the block reward will be `150` ODX (rounded down from `154.3`)
This does not include the current `20` ODN (per minute) built in inflation.  
The formula is `(supply/ratio)/time = block reward` or `(108177206/2)/350400 = 154.3 -> 150`  

After block 350,400 the reward will go back to 20 ODX, this means in practice less inflation as block time is now reduced to 3 min. 
As was pointed out by blackstone a high reward incentives staking, staking nodes are not effected by the inflation hit (a staking node with a reasonable network weight will make up for the high inflation) with cold staking active on day 1 it will be safe to even stake on online vpc rigs.

**Future changes** to the block reward are also in mind by proposing future changes to the protocol and allowing miners to accept such proposal using bip9 block version signalling mechanism, this additional block reward can be used as an economic incentive tool. 

**Airdrop on stratis coin holders:**  
I propose airdropping 5% on STRAT coin holders form the lost **cryptopia** coins (about 2.5 mil ODX).
The benefits are for this, ODX is using the STRAT code base and this is a way to give back for the development effort.
It also brings over a big community of miners and devs that are very familiar with the code base.
This will also make ODX become a known and accepted coin in the stratis community.   

The remainder of the cryptopia hacked coins 2.5 mil will be burned.  

**Foundation coins**  
We propose that the initial devs (and foundation) will not have more then 20% of total supply, this means `10,817,720.6` in total.  
Both main devs will have 5% each in total (this includes what they already have) and 10% `5,408,860.3` will go to the foundation (multisig or controlled by dangeshony).  

Out of the `17,478,743.5` ODX foundation what's left after the distribution above will be burned or kept in a future joiners pool (see below).  

Foundation coins will not stake and be hit by inflation over time.  
This is a pledge by the current active devs, but we cannot guarantee that coins paid out as bounty form this coins will not be used for staking.  

**Future Joiners Pool**
Due to the nature of POS where one needs coins to stake, in order to allow new serious users to join the community and not have to go via an exchange (with no liquidity) we could keep the remainder of the foundation funds in such a pool.
Those new stakers will exchange BTC for ODX at current market price (the funds will belong to the foundation and stay in escrow)

### Breakdown:

To summarise after airdrop (2:1 ratio) total supply is 54m.

The % of allocation of funds
- 20% foundation and devs 10.8m 
- 5%   airdrop on STRAT 2.7m
- 75% airdrop on ODN 40.5m

Two years of increased block reward (15 ODX) to reach the previous total supply and favour active miners.
Reduce block time to 3 min

**Comment by dangershony** [comment]
A proposal by Buta (odx discord)

To avoid pumps on STRAT, the airdrop total will be split and happen in a few stages.
For example 900 ODX airdropped 3 times on STRAT in different points in time (every 1.5 month).

**Comment by thecrypt0hunter** [comment]
Have you considered smoothing the reduction in rewards over time i.e., x% reduction after y blocks. Therefore its not a big bang reduction after 24 months?

**Comment by dangershony** [comment]
I did consider two stages first year to stake 60% and the second year 40% of the total ratio slashing
that's an option indeed, but is there any economic benefit?

**Comment by thecrypt0hunter** [comment]
It's less of a shock to the miners if the reduction is smoother - we may lose miners and security when the 24 months ticks over.

**Comment by dangershony** [comment]
I have no objections to a smoother distribution, but I don't think its such a big issue, after the 2 years we go back to the previous distribution (20 per block) which is still quite profitable for miners.

If we did so we could disperse 60% on year one and split year to 30% first half and then 10%
